gatsby/src/components/Sidebar/index.js

```diff
@@ -2,12 +2,27 @@ import React from "react";
 import { graphql, StaticQuery, Link as GatsbyLink } from "gatsby";
 import styled from "styled-components";
 import { Text } from "grommet";
+import { axisThemeConfig } from "@centrifuge/axis-theme";
 
-import { List, Item } from "../List";
+import { List, Item as ListItem } from "../List";
+import { navLinkStyles } from "../Links";
 
-const Link = styled(GatsbyLink)`
-  line-height: 24px;
+const Item = styled(ListItem)`
+  font-size: 14px;
```

Comment:
Don't we have access to the theme here? I think it will be cool if we reuse theme settings and not hardcode values in css

Reply:
these items are not using Grommet, theyre custom but i think I can rewokr them to use it.